crypto/rand/randfile.c

```diff
@@ -280,8 +280,16 @@ const char *RAND_file_name(char *buf, size_t size)
     struct stat sb;
 #endif
 
+    /* Find the directory where the state file should go. */
+#ifdef OPENSSL_SYS_WINDOWS
+    if ((s = getenv("RANDFILE")) == NULL) {
+        s = getenv("TEMP");
+        if (s == NULL)
+            s = getenv("APPDATA");
```

Comment:
I'd drop the APPDATA bit. On my machine `%TEMP%` is `AppData\Local\Temp`, but `%APPDATA%` it `AppData\Roaming`. Having the random file default to a roaming location (which is what I assume this is), doesn't seem like a great idea.